config/routes.rb

```diff
@@ -70,5 +80,5 @@
   end
 
 
-  root 'home#index'
+  root 'teams#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.